pvw/launcher/config.json

```diff
@@ -32,7 +32,9 @@
   },
   "apps": {
     "visualizer": {
-      "cmd": ["${python_exec}",
+      "cmd": ["env", "PARAVIEW_LOG_EXECUTION_VERBOSITY=INFO",
+        "env", "PARAVIEW_LOG_RENDERING_VERBOSITY=INFO",
+        "${python_exec}",
         EXTRA_PVPYTHON_ARGS
```

Comment:
this doesn't look like json, is it meant to be commented?

Reply:
maybe it is showing it was deleted? 🤷

Reply:
Nope, I don't think this is truly a JSON file. I'm pretty sure you've noticed this before too :)
Note I didn't touch that line, I think it is red because the GitHub linter also agrees with you that it isn't JSON.
Internally there is some substitution going on to fix this behind the scenes that I don't touch.